AK/Array.h

```diff
@@ -109,6 +109,27 @@ constexpr static auto iota_array(T const offset = {})
     return Detail::integer_sequence_generate_array<T>(offset, MakeIntegerSequence<T, N>());
 }
 
+namespace Detail {
+
+template<typename T>
+template<size_t size>
+ALWAYS_INLINE constexpr Span<T>::Span(Array<T, size>& array)
```

Comment:
Does anything actually need these constructors?

`Array` has an implicit `operator Span` much like `Vector` does, and that should be sufficient. I removed these constructors locally and everything builds fine. I'd rather that than awkwardly having a `Span::Span` definition in `Array.h`.

Reply:
Huh, I can't find any, either. I'll remove them then, good catch! :D